lib/pdk/cli/util/changelog_generator.rb

```diff
@@ -0,0 +1,58 @@
+module PDK
+  module CLI
+    module Util
+      class ChangelogGenerator
+        def generate_changelog
+          retries = 0
+          result = nil
+          loop do
+            result = system('bundle exec rake changelog')
+            retries += 1
+            break if result || retries > 3
+          end
+
+          unless result
+            raise 'Error generating changelog'
+          end
+
+          output = File.read('CHANGELOG.md')
```

Comment:
Dangerous defaults.  What about line endings or encoding?

Reply:
Will fix this